### ISSUE TITLE
Automated e-mails: remove references to postal address

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#480]
EIS stopped collecting postal addresses for domain contacts on 1st of January 2017.  Aputomated emails still include references to postal address fields and if still exist in registry also include it in the emails.

Address must be removed from all the e-mail templates and the country data must be replaced with data from ident_cc.

Email messages that include postal addresses today are:
* registrant change started
* Application of approval for registrant change of domain
* Domains’ contact e-mail address(es) have changed

#243

**Comment by artur-beljajev** [comment]
@teadur @ratM1n "Address" model is not used anywhere. I need your confirmation that the tables "addresses" and "log_addresses" can be safely deleted (on prod).

**Comment by ratM1n** [comment]
yes, both this tables (addresses, log_addresses) are empty in live database.

**Comment by artur-beljajev** [comment]
Both country name (country and address country) are now displayed in both Estonian and English.

**Comment by artur-beljajev** [comment]
@vohmar Is it ok if new and old registrants see each other in "to" email header when registrant change confirmation is sent?

**Comment by artur-beljajev** [comment]
phone is missing

**Comment by teadur** [comment]
From testing persepctive it confuses things, atleast before the emails were a bit different so it was good indicator that right person got the right email from TO

**Comment by artur-beljajev** [comment]
Why do you want to manually test it if it is already covered by specs?

**Comment by teadur** [comment]
Okey let me rephrase it, from verification perspective its more clear if only one TO is included.

**Comment by artur-beljajev** [comment]
What is meant by "bit different"? There was exactly the same content.

**Comment by teadur** [comment]
then the behaviour has changed, it used to be that these emails werent identical.

**Comment by artur-beljajev** [comment]
https://prnt.sc/fan8gu

**Comment by artur-beljajev** [comment]
Just compared those 2 templates just in case, and there are the same still.

**Comment by vohmar** [comment]
Cannot test as no e-mail is sent in case of registrant e-mail address change.

OK for registrant change related e-mails.

**Comment by artur-beljajev** [comment]
ContactMailer still uses whitelist, therefore it's neede to be sure email is present in `whitelist_emails_for_staging` key in application.yml

**Comment by vohmar** [comment]
yes the problem was with the whitelist missing my test e-mail address. Works now with a new bug with listing admin cotact type in english version of the notification (everything is fine in estonian):

Domains affected by this update:
timo-3k.ee (Registrant, Tech contact)
timo-6k.ee (Registrant, Tech contact)
timo-4a.ee (Registrant, Tech contact)
timo-9k.ee (Registrant, Tech contact)
timo-delete2.ee (Registrant, {:repp_logs=>{:index=>{:title=>"REPP log", :reset_btn=>"Reset"}}, :contact_versions=>{:index=>{:reset_btn=>"Reset"}}, :blocked_domains=>{:index=>{:title=>"Blocked domains", :new_btn=>"New blocked domain", :reset_btn=>"Reset"}}, :domain_versions=>{:archive=>{:reset_btn=>"Reset"}}, :billing=>{:prices=>{:index=>{:title=>"Prices", :new_btn=>"New price", :not_found=>"No price found"}, :new=>{:title=>"New price"}, :create=>{:created=>"Price has been created"}, :edit=>{:title=>"Edit price", :expire_btn=>"Expire", :expire_btn_confirm=>"Are you sure you want to expire price?"}, :update=>{:updated=>"Price has been updated"}, :expire=>{:expired=>"Price has been expired"}, :form=>{:create_btn=>"Create price", :update_btn=>"Update price"}, :search_form=>{:search_btn=>"Search", :reset_btn=>"Reset"}}}, :epp_logs=>{:index=>{:title=>"EPP log", :reset_btn=>"Reset"}, :show=>{:title=>"EPP log"}}, :menu=>{:users=>"Users", :api_users=>"API users", :admin_users=>"Admin users", :prices=>"Prices", :archive=>"Archive", :domain_history=>"Domain history", :contact_history=>"Contact history", :zones=>"Zones", :blocked_domains=>"Blocked domains", :reserved_domains=>"Reserved domains", :epp_log=>"EPP log", :repp_log=>"REPP log", :que=>"Que"}, :admin_users=>{:index=>{:title=>"Admin users", :new_btn=>"New admin user"}, :new=>{:title=>"New admin user"}}, :registrars=>{:index=>{:new_btn=>"New registrar"}, :show=>{:new_api_use_btn=>"New API user", :active=>"Active", :api_users=>"API users"}, :create=>{:created=>"Registrar has been successfully created", :not_created=>"Unable to create registrar"}, :update=>{:updated=>"Registrar has been successfully updated", :not_updated=>"Unable to update registrar"}, :form=>{:misc=>"Miscellaneous", :create_btn=>"Create registrar", :update_btn=>"Update registrar"}}, :api_users=>{:index=>{:new_btn=>"New API user", :title=>"API users", :active=>"Active"}, :show=>{:active=>"Active", :subject=>"Subject"}, :new=>{:title=>"New API user"}, :form=>{:active=>"Active"}}, :dns=>{:zones=>{:index=>{:title=>"Zones", :new_btn=>"New zone", :not_found=>"No zone found"}, :new=>{:title=>"New zone"}, :create=>{:created=>"Zone has been created"}, :edit=>{:title=>"Edit zone"}, :update=>{:updated=>"Zone has been updated"}, :form=>{:create_btn=>"Create zone", :update_btn=>"Update zone"}, :zone=>{:generate_zone_file_btn=>"Generate zone file"}}}, :contacts=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:new_status_btn=>"Add new status", :back_btn=>"Back to contact"}, :partials=>{:general=>{:created=>"Created", :updated=>"Updated"}, :domains=>{:roles=>"Roles"}}}, :reserved_domains=>{:index=>{:title=>"Reserved domains", :new_btn=>"New reserved domain", :reset_btn=>"Reset"}}, :account_activities=>{:index=>{:reset_btn=>"Reset"}}, :domains=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:add_new_status_btn=>"Add new status", :back_btn=>"Back to domain"}, :force_delete_dialog=>{:title=>"Force delete", :template=>"Template", :close_btn=>"Close dialog", :submit_btn=>"Force delete domain"}, :schedule_force_delete=>{:scheduled=>"Force delete procedure has been scheduled"}, :cancel_force_delete=>{:cancelled=>"Force delete procedure has been cancelled"}, :versions=>{:time=>"Time", :admin_contact=>"Admin contact"}, :partials=>{:general=>{:outzone_time=>"Outzone time", :delete_time=>"Delete time", :force_delete_time=>"Force delete time"}}, :force_delete_toggle_btn=>{:schedule=>"Schedule force delete", :cancel=>"Cancel force delete", :cancel_confim=>"Are you sure you want cancel force delete procedure?"}}}, Tech contact)
timo-tühik.ee (Registrant, {:repp_logs=>{:index=>{:title=>"REPP log", :reset_btn=>"Reset"}}, :contact_versions=>{:index=>{:reset_btn=>"Reset"}}, :blocked_domains=>{:index=>{:title=>"Blocked domains", :new_btn=>"New blocked domain", :reset_btn=>"Reset"}}, :domain_versions=>{:archive=>{:reset_btn=>"Reset"}}, :billing=>{:prices=>{:index=>{:title=>"Prices", :new_btn=>"New price", :not_found=>"No price found"}, :new=>{:title=>"New price"}, :create=>{:created=>"Price has been created"}, :edit=>{:title=>"Edit price", :expire_btn=>"Expire", :expire_btn_confirm=>"Are you sure you want to expire price?"}, :update=>{:updated=>"Price has been updated"}, :expire=>{:expired=>"Price has been expired"}, :form=>{:create_btn=>"Create price", :update_btn=>"Update price"}, :search_form=>{:search_btn=>"Search", :reset_btn=>"Reset"}}}, :epp_logs=>{:index=>{:title=>"EPP log", :reset_btn=>"Reset"}, :show=>{:title=>"EPP log"}}, :menu=>{:users=>"Users", :api_users=>"API users", :admin_users=>"Admin users", :prices=>"Prices", :archive=>"Archive", :domain_history=>"Domain history", :contact_history=>"Contact history", :zones=>"Zones", :blocked_domains=>"Blocked domains", :reserved_domains=>"Reserved domains", :epp_log=>"EPP log", :repp_log=>"REPP log", :que=>"Que"}, :admin_users=>{:index=>{:title=>"Admin users", :new_btn=>"New admin user"}, :new=>{:title=>"New admin user"}}, :registrars=>{:index=>{:new_btn=>"New registrar"}, :show=>{:new_api_use_btn=>"New API user", :active=>"Active", :api_users=>"API users"}, :create=>{:created=>"Registrar has been successfully created", :not_created=>"Unable to create registrar"}, :update=>{:updated=>"Registrar has been successfully updated", :not_updated=>"Unable to update registrar"}, :form=>{:misc=>"Miscellaneous", :create_btn=>"Create registrar", :update_btn=>"Update registrar"}}, :api_users=>{:index=>{:new_btn=>"New API user", :title=>"API users", :active=>"Active"}, :show=>{:active=>"Active", :subject=>"Subject"}, :new=>{:title=>"New API user"}, :form=>{:active=>"Active"}}, :dns=>{:zones=>{:index=>{:title=>"Zones", :new_btn=>"New zone", :not_found=>"No zone found"}, :new=>{:title=>"New zone"}, :create=>{:created=>"Zone has been created"}, :edit=>{:title=>"Edit zone"}, :update=>{:updated=>"Zone has been updated"}, :form=>{:create_btn=>"Create zone", :update_btn=>"Update zone"}, :zone=>{:generate_zone_file_btn=>"Generate zone file"}}}, :contacts=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:new_status_btn=>"Add new status", :back_btn=>"Back to contact"}, :partials=>{:general=>{:created=>"Created", :updated=>"Updated"}, :domains=>{:roles=>"Roles"}}}, :reserved_domains=>{:index=>{:title=>"Reserved domains", :new_btn=>"New reserved domain", :reset_btn=>"Reset"}}, :account_activities=>{:index=>{:reset_btn=>"Reset"}}, :domains=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:add_new_status_btn=>"Add new status", :back_btn=>"Back to domain"}, :force_delete_dialog=>{:title=>"Force delete", :template=>"Template", :close_btn=>"Close dialog", :submit_btn=>"Force delete domain"}, :schedule_force_delete=>{:scheduled=>"Force delete procedure has been scheduled"}, :cancel_force_delete=>{:cancelled=>"Force delete procedure has been cancelled"}, :versions=>{:time=>"Time", :admin_contact=>"Admin contact"}, :partials=>{:general=>{:outzone_time=>"Outzone time", :delete_time=>"Delete time", :force_delete_time=>"Force delete time"}}, :force_delete_toggle_btn=>{:schedule=>"Schedule force delete", :cancel=>"Cancel force delete", :cancel_confim=>"Are you sure you want cancel force delete procedure?"}}}, Tech contact)
110.ee (Registrant, {:repp_logs=>{:index=>{:title=>"REPP log", :reset_btn=>"Reset"}}, :contact_versions=>{:index=>{:reset_btn=>"Reset"}}, :blocked_domains=>{:index=>{:title=>"Blocked domains", :new_btn=>"New blocked domain", :reset_btn=>"Reset"}}, :domain_versions=>{:archive=>{:reset_btn=>"Reset"}}, :billing=>{:prices=>{:index=>{:title=>"Prices", :new_btn=>"New price", :not_found=>"No price found"}, :new=>{:title=>"New price"}, :create=>{:created=>"Price has been created"}, :edit=>{:title=>"Edit price", :expire_btn=>"Expire", :expire_btn_confirm=>"Are you sure you want to expire price?"}, :update=>{:updated=>"Price has been updated"}, :expire=>{:expired=>"Price has been expired"}, :form=>{:create_btn=>"Create price", :update_btn=>"Update price"}, :search_form=>{:search_btn=>"Search", :reset_btn=>"Reset"}}}, :epp_logs=>{:index=>{:title=>"EPP log", :reset_btn=>"Reset"}, :show=>{:title=>"EPP log"}}, :menu=>{:users=>"Users", :api_users=>"API users", :admin_users=>"Admin users", :prices=>"Prices", :archive=>"Archive", :domain_history=>"Domain history", :contact_history=>"Contact history", :zones=>"Zones", :blocked_domains=>"Blocked domains", :reserved_domains=>"Reserved domains", :epp_log=>"EPP log", :repp_log=>"REPP log", :que=>"Que"}, :admin_users=>{:index=>{:title=>"Admin users", :new_btn=>"New admin user"}, :new=>{:title=>"New admin user"}}, :registrars=>{:index=>{:new_btn=>"New registrar"}, :show=>{:new_api_use_btn=>"New API user", :active=>"Active", :api_users=>"API users"}, :create=>{:created=>"Registrar has been successfully created", :not_created=>"Unable to create registrar"}, :update=>{:updated=>"Registrar has been successfully updated", :not_updated=>"Unable to update registrar"}, :form=>{:misc=>"Miscellaneous", :create_btn=>"Create registrar", :update_btn=>"Update registrar"}}, :api_users=>{:index=>{:new_btn=>"New API user", :title=>"API users", :active=>"Active"}, :show=>{:active=>"Active", :subject=>"Subject"}, :new=>{:title=>"New API user"}, :form=>{:active=>"Active"}}, :dns=>{:zones=>{:index=>{:title=>"Zones", :new_btn=>"New zone", :not_found=>"No zone found"}, :new=>{:title=>"New zone"}, :create=>{:created=>"Zone has been created"}, :edit=>{:title=>"Edit zone"}, :update=>{:updated=>"Zone has been updated"}, :form=>{:create_btn=>"Create zone", :update_btn=>"Update zone"}, :zone=>{:generate_zone_file_btn=>"Generate zone file"}}}, :contacts=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:new_status_btn=>"Add new status", :back_btn=>"Back to contact"}, :partials=>{:general=>{:created=>"Created", :updated=>"Updated"}, :domains=>{:roles=>"Roles"}}}, :reserved_domains=>{:index=>{:title=>"Reserved domains", :new_btn=>"New reserved domain", :reset_btn=>"Reset"}}, :account_activities=>{:index=>{:reset_btn=>"Reset"}}, :domains=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:add_new_status_btn=>"Add new status", :back_btn=>"Back to domain"}, :force_delete_dialog=>{:title=>"Force delete", :template=>"Template", :close_btn=>"Close dialog", :submit_btn=>"Force delete domain"}, :schedule_force_delete=>{:scheduled=>"Force delete procedure has been scheduled"}, :cancel_force_delete=>{:cancelled=>"Force delete procedure has been cancelled"}, :versions=>{:time=>"Time", :admin_contact=>"Admin contact"}, :partials=>{:general=>{:outzone_time=>"Outzone time", :delete_time=>"Delete time", :force_delete_time=>"Force delete time"}}, :force_delete_toggle_btn=>{:schedule=>"Schedule force delete", :cancel=>"Cancel force delete", :cancel_confim=>"Are you sure you want cancel force delete procedure?"}}}, Tech contact)
kas-price-expire-toimib.ee (Registrant, {:repp_logs=>{:index=>{:title=>"REPP log", :reset_btn=>"Reset"}}, :contact_versions=>{:index=>{:reset_btn=>"Reset"}}, :blocked_domains=>{:index=>{:title=>"Blocked domains", :new_btn=>"New blocked domain", :reset_btn=>"Reset"}}, :domain_versions=>{:archive=>{:reset_btn=>"Reset"}}, :billing=>{:prices=>{:index=>{:title=>"Prices", :new_btn=>"New price", :not_found=>"No price found"}, :new=>{:title=>"New price"}, :create=>{:created=>"Price has been created"}, :edit=>{:title=>"Edit price", :expire_btn=>"Expire", :expire_btn_confirm=>"Are you sure you want to expire price?"}, :update=>{:updated=>"Price has been updated"}, :expire=>{:expired=>"Price has been expired"}, :form=>{:create_btn=>"Create price", :update_btn=>"Update price"}, :search_form=>{:search_btn=>"Search", :reset_btn=>"Reset"}}}, :epp_logs=>{:index=>{:title=>"EPP log", :reset_btn=>"Reset"}, :show=>{:title=>"EPP log"}}, :menu=>{:users=>"Users", :api_users=>"API users", :admin_users=>"Admin users", :prices=>"Prices", :archive=>"Archive", :domain_history=>"Domain history", :contact_history=>"Contact history", :zones=>"Zones", :blocked_domains=>"Blocked domains", :reserved_domains=>"Reserved domains", :epp_log=>"EPP log", :repp_log=>"REPP log", :que=>"Que"}, :admin_users=>{:index=>{:title=>"Admin users", :new_btn=>"New admin user"}, :new=>{:title=>"New admin user"}}, :registrars=>{:index=>{:new_btn=>"New registrar"}, :show=>{:new_api_use_btn=>"New API user", :active=>"Active", :api_users=>"API users"}, :create=>{:created=>"Registrar has been successfully created", :not_created=>"Unable to create registrar"}, :update=>{:updated=>"Registrar has been successfully updated", :not_updated=>"Unable to update registrar"}, :form=>{:misc=>"Miscellaneous", :create_btn=>"Create registrar", :update_btn=>"Update registrar"}}, :api_users=>{:index=>{:new_btn=>"New API user", :title=>"API users", :active=>"Active"}, :show=>{:active=>"Active", :subject=>"Subject"}, :new=>{:title=>"New API user"}, :form=>{:active=>"Active"}}, :dns=>{:zones=>{:index=>{:title=>"Zones", :new_btn=>"New zone", :not_found=>"No zone found"}, :new=>{:title=>"New zone"}, :create=>{:created=>"Zone has been created"}, :edit=>{:title=>"Edit zone"}, :update=>{:updated=>"Zone has been updated"}, :form=>{:create_btn=>"Create zone", :update_btn=>"Update zone"}, :zone=>{:generate_zone_file_btn=>"Generate zone file"}}}, :contacts=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:new_status_btn=>"Add new status", :back_btn=>"Back to contact"}, :partials=>{:general=>{:created=>"Created", :updated=>"Updated"}, :domains=>{:roles=>"Roles"}}}, :reserved_domains=>{:index=>{:title=>"Reserved domains", :new_btn=>"New reserved domain", :reset_btn=>"Reset"}}, :account_activities=>{:index=>{:reset_btn=>"Reset"}}, :domains=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:add_new_status_btn=>"Add new status", :back_btn=>"Back to domain"}, :force_delete_dialog=>{:title=>"Force delete", :template=>"Template", :close_btn=>"Close dialog", :submit_btn=>"Force delete domain"}, :schedule_force_delete=>{:scheduled=>"Force delete procedure has been scheduled"}, :cancel_force_delete=>{:cancelled=>"Force delete procedure has been cancelled"}, :versions=>{:time=>"Time", :admin_contact=>"Admin contact"}, :partials=>{:general=>{:outzone_time=>"Outzone time", :delete_time=>"Delete time", :force_delete_time=>"Force delete time"}}, :force_delete_toggle_btn=>{:schedule=>"Schedule force delete", :cancel=>"Cancel force delete", :cancel_confim=>"Are you sure you want cancel force delete procedure?"}}}, Tech contact)
šž-test.ee (Registrant, {:repp_logs=>{:index=>{:title=>"REPP log", :reset_btn=>"Reset"}}, :contact_versions=>{:index=>{:reset_btn=>"Reset"}}, :blocked_domains=>{:index=>{:title=>"Blocked domains", :new_btn=>"New blocked domain", :reset_btn=>"Reset"}}, :domain_versions=>{:archive=>{:reset_btn=>"Reset"}}, :billing=>{:prices=>{:index=>{:title=>"Prices", :new_btn=>"New price", :not_found=>"No price found"}, :new=>{:title=>"New price"}, :create=>{:created=>"Price has been created"}, :edit=>{:title=>"Edit price", :expire_btn=>"Expire", :expire_btn_confirm=>"Are you sure you want to expire price?"}, :update=>{:updated=>"Price has been updated"}, :expire=>{:expired=>"Price has been expired"}, :form=>{:create_btn=>"Create price", :update_btn=>"Update price"}, :search_form=>{:search_btn=>"Search", :reset_btn=>"Reset"}}}, :epp_logs=>{:index=>{:title=>"EPP log", :reset_btn=>"Reset"}, :show=>{:title=>"EPP log"}}, :menu=>{:users=>"Users", :api_users=>"API users", :admin_users=>"Admin users", :prices=>"Prices", :archive=>"Archive", :domain_history=>"Domain history", :contact_history=>"Contact history", :zones=>"Zones", :blocked_domains=>"Blocked domains", :reserved_domains=>"Reserved domains", :epp_log=>"EPP log", :repp_log=>"REPP log", :que=>"Que"}, :admin_users=>{:index=>{:title=>"Admin users", :new_btn=>"New admin user"}, :new=>{:title=>"New admin user"}}, :registrars=>{:index=>{:new_btn=>"New registrar"}, :show=>{:new_api_use_btn=>"New API user", :active=>"Active", :api_users=>"API users"}, :create=>{:created=>"Registrar has been successfully created", :not_created=>"Unable to create registrar"}, :update=>{:updated=>"Registrar has been successfully updated", :not_updated=>"Unable to update registrar"}, :form=>{:misc=>"Miscellaneous", :create_btn=>"Create registrar", :update_btn=>"Update registrar"}}, :api_users=>{:index=>{:new_btn=>"New API user", :title=>"API users", :active=>"Active"}, :show=>{:active=>"Active", :subject=>"Subject"}, :new=>{:title=>"New API user"}, :form=>{:active=>"Active"}}, :dns=>{:zones=>{:index=>{:title=>"Zones", :new_btn=>"New zone", :not_found=>"No zone found"}, :new=>{:title=>"New zone"}, :create=>{:created=>"Zone has been created"}, :edit=>{:title=>"Edit zone"}, :update=>{:updated=>"Zone has been updated"}, :form=>{:create_btn=>"Create zone", :update_btn=>"Update zone"}, :zone=>{:generate_zone_file_btn=>"Generate zone file"}}}, :contacts=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:new_status_btn=>"Add new status", :back_btn=>"Back to contact"}, :partials=>{:general=>{:created=>"Created", :updated=>"Updated"}, :domains=>{:roles=>"Roles"}}}, :reserved_domains=>{:index=>{:title=>"Reserved domains", :new_btn=>"New reserved domain", :reset_btn=>"Reset"}}, :account_activities=>{:index=>{:reset_btn=>"Reset"}}, :domains=>{:index=>{:reset_btn=>"Reset"}, :edit=>{:add_new_status_btn=>"Add new status", :back_btn=>"Back to domain"}, :force_delete_dialog=>{:title=>"Force delete", :template=>"Template", :close_btn=>"Close dialog", :submit_btn=>"Force delete domain"}, :schedule_force_delete=>{:scheduled=>"Force delete procedure has been scheduled"}, :cancel_force_delete=>{:cancelled=>"Force delete procedure has been cancelled"}, :versions=>{:time=>"Time", :admin_contact=>"Admin contact"}, :partials=>{:general=>{:outzone_time=>"Outzone time", :delete_time=>"Delete time", :force_delete_time=>"Force delete time"}}, :force_delete_toggle_btn=>{:schedule=>"Schedule force delete", :cancel=>"Cancel force delete", :cancel_confim=>"Are you sure you want cancel force delete procedure?"}}}, Tech contact)
timoarturtest.ee (Registrant)
timo-contact.ee (Tech contact)

**Comment by artur-beljajev** [comment]
@teadur @ratM1n Please ensure `domains.registrar_id` and `registrant_id` exist in corresponding tables.

**Comment by vohmar** [comment]
html version of the email is broken in both languages:
```
Domains affected by this update:
timo-3k.ee (Registrant, Technical contact)<br>timo-6k.ee (Registrant, Technical contact)<br>timo-4a.ee (Registrant, Technical contact)<br>timo-9k.ee (Registrant, Technical contact)<br>timo-delete2.ee (Registrant, Administrative contact, Technical contact)<br>timo-tühik.ee (Registrant, Administrative contact, Technical contact)<br>110.ee (Registrant, Administrative contact, Technical contact)<br>kas-price-expire-toimib.ee (Registrant, Administrative contact, Technical contact)<br>šž-test.ee (Registrant, Administrative contact, Technical contact)<br>timoarturtest.ee (Registrant)<br>timo-contact.ee (Technical contact)
```

**Comment by vohmar** [comment]
in production
